src/utils/allowCredentialsInHeader.js

```diff
@@ -0,0 +1,5 @@
+import axios from "axios";
+
```

Comment:
@lazycipher now you are not using authorization header ?
Only sending cookie?

Reply:
Yes after shifting the auth method to use cookies, we're using auth in the cookie header.